nilearn/plotting/data/html/report_body_template.html

```diff
@@ -1,85 +1,81 @@
-<?xml version="1.0" encoding="utf-8" ?>
-<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
-<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
-
-<head>
-<title>HTML report</title>
-<meta name = "viewport" content = "width = device-width, initial-scale = 1">
+<!-- CSS for the report -->
 <link rel="stylesheet" href="https://unpkg.com/purecss@1.0.0/build/pure-min.css" integrity="sha384-nn4HPE8lTHyVtfCBi5yW9d20FjT8BJwUXyWZT9InLYax14RDjBj46LmSztkmNP9w" crossorigin="anonymous">
+<link rel="stylesheet" href="https://unpkg.com/purecss@1.0.0/build/grids-responsive-min.css">
 <style type="text/css">
 
-h1 { padding-top: 15px; }
+div.nilearn_report {
```

Comment:
The goal of prefixing everything with div.nilearn_report is to avoid CSS interferences, now that we are in the same HTML page.